bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
+      "answer": "NAND, NOR"
+    },
+    {
+      "id": 419,
+      "question": "What is the mathematical formalism which functional programming was built on?",
+      "answer": "lambda calculus"
+    },
+    {
+      "id": 420,
+      "question": "Why is a DDoS attack different from a DoS attack (answer in letters - A. because the victim's server was indefinitely disrupted from the amount of traffic, B. because it also attacks the victim's confidentiality, C. because the attack had political purposes behind it, D. because the traffic flooding the victim originated from many different sources)?",
+      "answer": "D"
+    },
+    {
+      "id": 421,
+      "question": "What is a HTTP/1.1 feature that was superseded by HTTP/2 multiplexing and is unsupported in most browsers nowadays?",
+      "answer": "pipelining"
+    },
+    {
+      "id": 422,
+      "question": "Which of these languages is the oldest? Tcl, Smalltalk 80, Haskell, Standard ML, Java",
+      "answer": "Smalltalk 80"
+    },
+    {
+      "id": 423,
+      "question": "What is the name for unicode codepoints that do not fit into 16 bits?",
+      "answer": "surrogates"
+    },
+    {
+      "id": 424,
+      "question": "Under what locale does making a string lowercase behave differently?",
+      "answer": "Turkish"
+    },
+    {
+      "id": 425,
+      "question": "What does the \"a\" represent in a HSLA color value?",
+      "answer": "transparency, translucency, alpha value, alpha channel"
+    },
+    {
+      "id": 426,
+      "question": "What is the section of a GIF that is limited to 256 colors called?",
+      "answer": "image block"
+    },
+    {
+      "id": 427,
+      "question": "What is an interpreter capable of interpreting itself called?",
+      "answer": "metainterpreter"
+    },
+    {
+      "id": 428,
+      "question": "Due to what data storage medium did old programming languages, such as cobol, ignore all characters past the 72nd column?",
+      "answer": "punch cards"
+    },
+    {
+      "id": 429,
+      "question": "Which of these sorting algorithms is not stable? Counting sort, quick sort, insertion sort, tim sort, bubble sort",
+      "answer": "quick, quick sort"
+    },
+    {
+      "id": 430,
+      "question": "Which of these languages is the youngest? Lisp, Python, Java, Haskell, Prolog, Ruby, Perl",
+      "answer": "Java"
+    }
+  ],
+  "python": [
+    {
+      "id": 501,
+      "question": "Is it all objects (y/n)?",
+      "answer": "y, yes"
+    },
+    {
+      "id": 502,
+      "question": "Name the only non-dunder method of the builtin slice object.",
+      "answer": "indices"
+    },
+    {
+      "id": 503,
+      "question": "What exception, other than `StopIteration`, can you raise from a `__getitem__` dunder to indicate to an iterator that it should stop?",
+      "answer": "IndexError"
+    },
+    {
+      "id": 504,
+      "question": "What type does the `&` operator return given 2 `dict_keys` objects?",
```

Comment:
This question tripped me up when first reading it because it seems a little ambiguous at first, I think "return when given 2..." would make it more understandable at a glance

Reply:
right, you gotta add 'em conjunctions